evennia/web/website/templates/website/help_list.html

```diff
@@ -0,0 +1,110 @@
+{% extends "base.html" %}
+
+{% block titleblock %}
+{{ view.page_title }}
+{% endblock %}
+
+{% block content %}
+
+{% load addclass %}
+<div class="row">
+  <div class="col">
+    <div class="card">
+      <div class="card-body">
+        <h1 class="card-title">{{ view.page_title }}</h1>
+        <hr />
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item"><a href="{% url 'help' %}">Compendium</a></li>
+        </ol>
+        <hr />
+        <div class="row">
+          {% regroup object_list by help_category as category_list %}
+          
+          {% if category_list %}
+          <!-- left column -->
+          <div class="col-lg-9 col-sm-12">
+            
+            <!-- intro -->
+            <div class="card border-light">
+              <div class="card-body">
+                <h4 class="card-title">The {{ game_name }} Compendium</h4>
+                <p>This section of the site is your guide to learning all about {{ game_name }}-- its rules, its setting, its people, customs, classes, castes, conflicts, races, religions, lore and more!</p>
```

Comment:
While enthusiastic, it's probably better to keep this default blurb more neutral and not mention just what detail you will learn about a particular game.

Reply:
I was having trouble coming up with a more generic blurb, but not putting one at all left the page feeling rather sterile. I'm open to suggestions...!